contrib/bsp/src/mill/contrib/bsp/MillBuildServer.scala

```diff
@@ -154,17 +154,17 @@ class MillBuildServer(evaluator: Evaluator,
       for (targetId <- dependencySourcesParams.getTargets.asScala) {
         val millModule = targetIdToModule(targetId)
         var sources = evaluateInformativeTask(evaluator,
-                                              millModule.resolveDeps(millModule.transitiveIvyDeps),
+                                              millModule.resolveDeps(millModule.transitiveIvyDeps, true),
```

Comment:
Isn't this resolving source jars instead of binary jars?

Reply:
Oh, now I see, the `var` already has the name `sources`, so indeed, the patch might just do fix something.